ETWProviders/etwprof.cpp

```diff
@@ -120,28 +120,35 @@ class CETWRegister
 } g_ETWRegister;
 
 // Redirector function for EventRegister. Called by macros in ETWProviderGenerated.h
-ULONG EVNTAPI EventRegister( LPCGUID ProviderId, PENABLECALLBACK EnableCallback, PVOID CallbackContext, PREGHANDLE RegHandle )
+#pragma warning(suppress: 28253)
+_Success_(return == ERROR_SUCCESS)

```

Comment:
This annotation doesn't seem to be working, and isn't present in Microsoft's definition of this function.

Why add the annotation if it doesn't seem to work (i.e.; warning C6101).